src/Symfony/Component/Routing/Exception/MissingMandatoryParametersException.php

```diff
@@ -19,4 +19,28 @@
  */
 class MissingMandatoryParametersException extends \InvalidArgumentException implements ExceptionInterface
 {
+    private array $missingParameters;
+    private string $routeName;
+
+    public function __construct(array $missingParameters, string $routeName, int $code = 0, \Throwable $previous = null)
```

Comment:
Adding mandatory parameters to the constructor is changing the signature in breaking way, they could be nullable. We need to preserve the old behavior also, which is to pass a message string as first arg. 
See e.g. https://github.com/symfony/symfony/pull/42420/files#diff-f95930cc6775d5eb314ac68373c2323eb599e74c140f6e665bb106b3b513f42cR44-R48 for inspiration

Reply:
Sorry about that, I fixed accordingly, what do you think ?

Reply:
Looks good! thanks